.circleci/config.yml

```diff
@@ -114,7 +114,7 @@ jobs:
         type: enum
         enum: ['x64', 'arm64']
     executor: << parameters.executor >>
-    parallelism: 2
+    parallelism: 1
```

Comment:
This was needed due to limiting of parallelism in our current CircleCI plan.

Reply:
Opened into a separate PR https://github.com/electron/forge/pull/3798

Reply:
I think we should document this in a comment, for if we ever have the means to run it in parallel again